### PR TITLE
taboola json changes

### PR DESCRIPTION
**Pull request by talivneh**
made changes in Taboola detection

### CODE REVIEW

File: src/technologies/t.json [hunk 1]
```diff
@@ -335,9 +335,13 @@
     "icon": "Taboola.svg",
     "js": {
       "_taboola": "",
-      "_taboolaNetworkMode": "",
+      "TRCImpl": "",
+      "TRC": "",
```
Comment (nurbek91) [inline src/technologies/t.json:339]: Hey mate, thanks for finding a bug!!!
I would advise adding an additional check to reduce some of the possible false positives.
examples:
https://www.morhipo.com/
http://thestartmagazine.com/feed/summary
```
"js": {
  "_taboola": "",
  "_taboolaNetworkMode": "",
  "taboola_view_id": ""
  "TRC.cdsPips": "",
  "trc_item_ur"l: "//taboolanews\\.com/",
  "TRCImpl.global": ""
},
```

Comment (talivneh) [reply src/technologies/t.json]: added TRCImpl.global, the others doesn't necessarily exists.

Comment (talivneh) [reply src/technologies/t.json]: I removed TRCImpl  and TRC. There is no property that is necessarily on the TRC.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**